.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,13 +6,24 @@ labels: ''
 assignees: ''
 
 ---
+<!-- Please search existing issues for potential duplicates before filing yours:	
+https://github.com/godotengine/godot/issues?q=is%3Aissue	
+-->
 
 **Godot version:**
+<!-- Specify commit hash if non-official. -->
+
 
 **OS/device including version:**
+<!-- Specify GPU model and drivers if graphics-related. -->
+
 
 **Issue description:**
+<!-- What happened, and what was expected. -->
+
 
 **Steps to reproduce:**
 
+
 **Minimal reproduction project:**
+<!-- Recommended as it greatly speeds up debugging. Drag and drop a zip archive to upload it. -->
```

Comment:
This doesn't really explain what IS a reproduction project. How about something like
"A small Godot project that reproduces the issue. Drag and drop a zip archive to upload it."

Reply:
Yeah, I think we can omit the "Recommended" to make sure people do upload the minimal projects.